plugins/cilium-cni/cni-install.sh

```diff
@@ -24,6 +24,8 @@ esac
 
 ENABLE_DEBUG=false
 CNI_EXCLUSIVE=true
+LOG_FILE=""
```

Comment:
I am on the fence on the default value (i.e. disable by default, or enable with default path /run/cilium/cilium-cni.log)

Reply:
It should be enabled by default with the default directory being `/run/cilium/cilium-cni.log`. Just a small note that `cilium-cni` should not crash /stop working if we can't create files in this default directory.

Reply:
thanks, both directories are actually the same due to soft link. I updated the path to /var/run/cilium/cilium-cni.log for consistency.

Reply:
I have tested with some invalid path (e.g. /var/invalid/path/cilium-cni.log), the log is not created, but connectivity test is passed as usual.